content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -376,10 +397,8 @@ IngressClass is marked as default in your cluster.
 {{< /caution >}}
 
 There are some ingress controllers, that work without the definition of a
-default `IngressClass`. For example, the Ingress-NGINX controller can be
-configured with a [flag](https://kubernetes.github.io/ingress-nginx/#what-is-the-flag-watch-ingress-without-class)
-`--watch-ingress-without-class`. It is [recommended](https://kubernetes.github.io/ingress-nginx/#i-have-only-one-instance-of-the-ingresss-nginx-controller-in-my-cluster-what-should-i-do)  though, to specify the
-default `IngressClass`:
+default IngressClass. It is recommended though, to specify the default
```

Comment:
🤔 Should we just drop the first sentence here?

> When using Ingress, start by setting the default IngressClass:

(If we keep it, the leftover comma after "controllers" needs to go. 🙂)